packages/cxl-ui/src/components/cxl-institute-layout.js

```diff
@@ -0,0 +1,66 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlInstituteLayoutStyles from '../styles/cxl-institute-layout-css.js';
+
+import '@vaadin/vaadin-tabs';
+
+class CXLInstituteLayoutElement extends LitElement {
+  static get properties() {
+    return {};
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlInstituteLayoutStyles];
+  }
+
+  render() {
+    return html`
+      <!-- Visible on mobile -->
+      <header id="header" part="header">
+        <a href="/" class="logo">
+          <span class="cxl">C<span>XL</span></span>
+        </a>
+        <h3>Something something</h3>
+      </header>
+
+      <!-- Visible on mobile -->
+      <vaadin-tabs
+        id="tabs"
+        part="tabs"
+        theme="equal-width-tabs cxl-institute-layout-tabs"
+        selected="1"
+      >
+        <vaadin-tab @click="${() => this.changeTab('sidebar')}">Menu</vaadin-tab>
+        <vaadin-tab @click="${() => this.changeTab('content')}">Something here</vaadin-tab>
+      </vaadin-tabs>
+
+      <div part="icon-nav">
+        <slot name="icon-nav"></slot>
+      </div>
+
+      <div id="sidebar" part="sidebar">
+        <div class="sidebar-header">
+          <slot name="sidebar-header"></slot>
+        </div>
+
+        <slot name="sidebar-content"></slot>
+      </div>
+
+      <div id="content" part="content" page-visible>
```

Comment:
@chanar OK with you to drop `id` attributes, and continue w/ `part`?

Seems to be Vaadin model, right? Since why would you do ID-s inside Shadow DOM.

Reply:
Got confirmation from Vaadin https://gitter.im/vaadin/web-components?at=5cdbf2795a1d435d4645d524 so dropping ID-s where not specifically needed.